pkg/kubectl/resource_printer.go

```diff
@@ -786,8 +788,17 @@ func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 			return err
 		}
 	}
+
+	podColor := "<fg 2>" // green
+	if readyContainers != totalContainers {
+		podColor = "<fg 11>" // yellow
+	}
+	if readyContainers == 0 {
+		podColor = "<fg 1>" // red
```

Comment:
@brendandburns 
> I worry that we'll end up with lots of different colors if we don't focus on coloring the entire row a single shade...

Based on this comment, and @smarterclayton's feedback, perhaps we could keep red for rows where the pod status is failing, and no color / how it is currently for any other case. WDYT?